src/batch-builder.js

```diff
@@ -1370,46 +1365,6 @@ module.exports = class BatchBuilder {
         return  finalStr;
     }
 
-    /**
-     * Return nops tx data-availability
-     * fromIdx | toIdx | amountF | userFee == 0 | 0 | 0 | 0
-     * @return {String} nop tx data availability encoded as hexadecimal
-     */
-    _nopTxsData() {
```

Comment:
I would not delete these functions since they could be useful for further data retrieving or testing

Reply:
Yes. Functions restored in next commit.